sycamore/tests/unit/scans/test_binary_scan.py

```diff
@@ -17,3 +21,29 @@ def test_partition(self):
             "parent_id",
             "properties",
         ]
+
+    def test_json_manifest(self):
+        base_path = str(TEST_DIR / "resources/data/htmls/")
+        remote_url = "https://en.wikipedia.org/wiki/Binary_search_algorithm"
+        indexed_at = "2023-10-04"
+        manifest = {
+            base_path + "/wikipedia_binary_search.html": {"remote_url": remote_url, "indexed_at": indexed_at},
+            "other file.html": {"remote_url": "value", "indexed_at": "date"},
+            "non-dict element": {"key1": "value1", "key2": ["listItem1", "listItem2"]},
+            "list property": ["listItem1", "listItem2"],
+        }
+        tmp_manifest = tempfile.NamedTemporaryFile(mode="w+")
```

Comment:
More of a convenience, but you can use the tmp_path fixture in pytest if you want to avoid having to manage this yourself. https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Nicee! Let me add this in the follow up where I create system properties